openmls/src/group/core_group/staged_commit.rs

```diff
@@ -384,6 +384,14 @@ impl StagedCommit {
         }
     }
 
+    #[cfg(test)]
```

Comment:
You don't need this method. It only has 1 use site and can be replaced by `add_proposals`